tamr_unify_client/models/project/resource.py

```diff
@@ -78,6 +78,25 @@ def as_mastering(self):
             )
         return MasteringProject(self.client, self._data, self.api_path)
 
+    def add_source_dataset(self, dataset):
+        """
+        Associate a dataset with a project in Unify.
+
+        By default, datasets are not associated with any projects.
+        They need to be added as input to a project before they can be used
+        as part of that project
+
+        :param project: Unify Project
+        :param dataset: Unify Dataset
+        :return: HTTP response from the server
+        :rtype: :class:`requests.models.Response`
```

Comment:
You can just do:

```suggestion
        :rtype: :class:`requests.Response`
```

See [this example in our codebase](https://github.com/Datatamer/unify-client-python/blob/a5606e571ac6058cd93cb8ac60482e55ce444739/tamr_unify_client/client.py#L98).

Reply:
Also, if you haven't already, you should run `make html` from within the `docs` directory and open the built HTML locally to check that the docs look correct